samples/powerpoint/shapes/binding-to-shapes.yaml

```diff
@@ -0,0 +1,370 @@
+id: powerpoint-shapes-binding-to-shapes
+name: Binding to shapes
+description: Shows how to create binding references for images and work with z-order.
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.8'
+script:
+    content: |
+        document
+          .getElementById("insert-image-and-create-binding")
+          .addEventListener("click", () => tryCatch(insertImageWithBinding));
+        document.getElementById("replace-binding1").addEventListener("click", () => tryCatch(replaceImageinBinding1));
```

Comment:
You can get rid of the 1..3 functions by using this syntax to pass the values instead: 
```typescript
document.getElementById("replace-binding1").addEventListener("click", () => tryCatch(() => {replaceBinding(1)}));
document.getElementById("replace-binding2").addEventListener("click", () => tryCatch(() => { replaceBinding(2) }));
document.getElementById("replace-binding3").addEventListener("click", () => tryCatch(() => { replaceBinding(3) }));
```